source/beamline/BioXAS/BioXASMainBeamline.h

```diff
@@ -57,12 +58,30 @@ class BioXASMainBeamline : public AMBeamline
 
 	/// Destructor.
 	virtual ~BioXASMainBeamline();
-
+    /// Returns true if all beamline components are connected, false otherwise.
```

Comment:
There are two places in creator that you need to enable that option.

Preferences->TextEditor->Behavior

and

Preferences->C++->CodeStyle->Edit->General

You might need to make a new profile for those C++ options.